lib/validator/validator.go

```diff
@@ -34,7 +34,7 @@ var validate *validator.Validate
 var (
 	nameRegex          = regexp.MustCompile("^[a-zA-Z0-9_.-]{1,128}$")
 	interfaceRegex     = regexp.MustCompile("^[a-zA-Z0-9_-]{1,15}$")
-	labelRegex         = regexp.MustCompile("^([a-zA-Z0-9.-]{0,253}/)?[a-zA-Z0-9]([a-zA-Z0-9_.-]{0,61}[a-zA-Z0-9])?$")
+	labelRegex         = regexp.MustCompile(`^([a-zA-Z0-9\_\.\-\/]{0,252}[a-zA-Z0-9])?$`)
```

Comment:
> Why did you change from quotes to backticks?

because you get a compile time error with some of those escape chars, so have to use runes.

> Also, do we really need to escape all of those?

yeah the only one that could have been without escape was `/` but I did it to be consistent.

> Also, why the switch to 252 characters?

so it's 252 (alphanumeric+specials) + 1 only alphanumeric so it doesn't end with a special char